### PR TITLE
Fixed build errors

### PR DESCRIPTION
**Pull request by cssuh**
Fixes build errors similar to this: 
`Argument of type 'string | TreeItemLabel' is not assignable to parameter of type 'string'.
  Type 'TreeItemLabel' is not assignable to type 'string'`

### CODE REVIEW

**Comment by alanrenmsft** [comment]
are you sure the the label is always string? @cssuh

**Comment by cssuh** [comment]
Yes, `clipboardWriteText` only accepts a string parameter, same with `toLowerCase`, `TreeNodeInfo` first parameter of the constructor should be a string, and `historyNodeLabel` should return a string.  Unless there is something I'm missing I believe they should all be strings.

File: src/objectExplorer/objectExplorerService.ts [hunk 1]
```diff
@@ -229,7 +229,15 @@ export class ObjectExplorerService {
      */
     public sortByServerName(array: TreeNodeInfo[]): TreeNodeInfo[] {
         const sortedNodeArray = array.sort((a, b) => {
-            return (a.label as string).toLowerCase().localeCompare((b.label as string).toLowerCase());
+            if (typeof a.label === 'string' && typeof b.label === 'string') {
```
Comment (alanrenmsft) [inline src/objectExplorer/objectExplorerService.ts:232]: to simplify the code, I would first get the string value for a and b and then do the compare

File: src/objectExplorer/objectExplorerService.ts [hunk 2]
```diff
@@ -484,10 +492,17 @@ export class ObjectExplorerService {
             if (!(<IConnectionProfile>node.connectionCredentials).savePassword) {
                 node.connectionCredentials.password = '';
             }
+            let disconnectedNode;
             // make a new node to show disconnected behavior
-            let disconnectedNode = new TreeNodeInfo(node.label as string, Constants.disconnectedServerLabel,
-                node.collapsibleState, node.nodePath, node.nodeStatus, Constants.disconnectedServerLabel,
-                undefined, node.connectionCredentials, node.parentNode);
+            if (typeof node.label === 'string') {
```
Comment (alanrenmsft) [inline src/objectExplorer/objectExplorerService.ts:497]: same here:
`
const label = typeof node.label === 'string' ? node.label:node.label.label;
`

Comment (alanrenmsft) [reply src/objectExplorer/objectExplorerService.ts:497]: giving the fact that it is being used so many times, you can create a util function to get the label from a node.

Comment (cssuh) [reply src/objectExplorer/objectExplorerService.ts:497]: ok that makes a lot of sense, I knew there had to be a simpler way

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**